lib/dialects/mssql/query-generator.js

```diff
@@ -570,7 +570,14 @@ module.exports = (function() {
 
       if (options.limit || options.offset) {
         if (!options.order || (options.include && !subQueryOrder.length)) {
-          fragment += ' ORDER BY ' + this.quoteIdentifier(model.primaryKeyAttribute);
+          if(!isSubQuery && options.order) {
+            fragment += ', ';
```

Comment:
ah okay, sorry I'm sort of acting mssql-maintainer-in-absentia, so excuse my lack of context as I'm not explicitly writing these queries. It would be really helpful if you could add contextual clues into the commit description in the future. Thanks for the contribution!